### PR TITLE
Standardization

### PR DESCRIPTION
**Pull request by WestonVincze**
PR for standardization

### CODE REVIEW

File: libs/react-numerics/tsconfig.json [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
-    "noPropertyAccessFromIndexSignature": true,
-    "noFallthroughCasesInSwitch": true
+    "isolatedModules": true,
+    "noFallthroughCasesInSwitch": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true
   },
   "files": [],
-  "include": [],
+  "include": ["@types", "src"],
```
Comment (rlmcneary2) [inline libs/react-numerics/tsconfig.json:24]: I'm not familiar with all these options but do they affect the ability build and publish?

These defaults are set by NX during setup. Does this break any of the NX scripts (I wouldn't expect them to)?

I don't understand, why is `strict` false?

File: libs/react-numerics/README.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ formatted numbers such as currency or telephone numbers.
 
 A [Bitovi React team](https://www.bitovi.com/frontend-javascript-consulting/react-consulting) project.
 
+## Need help or have questions?
```
Comment (rlmcneary2) [inline libs/react-numerics/README.md:8]: If the docs aren't linked to by the package.json `homepage` then I think the first line in this help section should be a link to the documentation.

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+16
```
Comment (rlmcneary2) [inline .nvmrc:1]: Thank you!

**Review by rlmcneary2 — CHANGES_REQUESTED** [review]
Weston thanks for doing this, I do have some concerns. I don't know if you know all the answers or not. If you can let me know who I should talk to I'd appreciate it.

**Comment by christopherjbaker** [comment]
Closing for #27

**Pull request closed without merging**